task/sonarqube-scanner/0.1/sonarqube-scanner.yaml

```
@@ -62,10 +73,16 @@ spec:
           if [[ -n "$(params.SONAR_PROJECT_KEY)" ]]; then
             replaceValues $(workspaces.source-dir.path)/sonar-project.properties sonar.projectKey $(params.SONAR_PROJECT_KEY)
           fi
+          if [[ -n "$(params.SONAR_LOGIN)" ]]; then
```

I guess this should be
```suggestion
          if [[ -n "$SONAR_LOGIN" ]]; then
```